drush/Commands/SanitizationCommands.php

```diff
@@ -195,4 +196,42 @@ private function getEntityStorage(string $type) {
     return $this->entityTypeManager->getStorage($type);
   }
 
+  /**
+   * Sanitize the database table for the user roles.
+   *
+   * @hook post-command sql-sanitize
+   */
+  public function userRole($result, CommandData $commandData) {
+    if (!$commandData->input()->getOption('sanitize-roles')) {
+      return;
+    }
+    // Remove all authors roles from the database.
+    $role = Database::getConnection()->truncate('user_roles')->execute();
+    if ($role) {
+      $this->logger()->error(dt('The user roles have not been truncated.'));
+    }
+    else {
+      $this->logger()->success(dt('The user roles have been truncated.'));
+    }
+  }
+
+  /**
+   * Sanitize the database table for the username.
+   *
+   * @hook post-command sql-sanitize
+   */
+  public function userName($result, CommandData $commandData) {
+    if (!$commandData->input()->getOption('sanitize-name')) {
+      return;
+    }
+    // User data table updated.
+    $name = Database::getConnection()->truncate('users_data table')->execute();
```

Comment:
User names are stored in `users_field_data`, not users_data. Also, we dont want to truncate that whole table, or else all our content will have no owner. I think we want a method very similar to https://github.com/drush-ops/drush/blob/40092d5dcf6f850fdf15d817c33602fe92c67c7c/src/Drupal/Commands/sql/SanitizeUserTableCommands.php#L26-L84. That sanitizes email addresses and passwords.

Reply:
I've updated the username sanitization option.